apps/web/src/routes/GroupSettings/DeleteSettingsTab/DeleteSettingsTab.js

```diff
@@ -1,41 +1,46 @@
 import PropTypes from 'prop-types'
 import React, { useCallback, useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
-import Button from 'components/Button'
+import { TriangleAlert } from 'lucide-react'
+import Button from 'components/ui/button'
 import { useViewHeader } from 'contexts/ViewHeaderContext'
 
-import classes from './DeleteSettingsTab.module.scss'
-
 function DeleteSettingsTab ({ group, deleteGroup }) {
   const { name } = group
   const { t } = useTranslation()
 
   const handleDeleteGroup = useCallback(() => {
-    if (window.confirm(t('Are you sure you want to delete the group {{name}}?', { name }))) {
+    if (window.confirm(t('Are you sure you want to delete the group {{name}}? This will permanently delete the group and all of its content. This cannot be undone.', { name }))) {
```

Comment:
So, this changes the 'key' to a translation, and then, since that new key doesn't exist, it will miss the key lookup and just display the key. So now it will no longer translate to spanish.

To change the copy, you need to make sure there is an entry in the locale files that matches the key. And you can change the values of those entries to the desired copy